Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -974,6 +985,17 @@ namespace AzToolsFramework
             m_forceAutoExpand = false; // Does not always expand by default
             reader.Read<bool>(m_forceAutoExpand);
         }
+        else if (attributeName == AZ::Edit::Attributes::CategoryStyle)
+        {
+            AZStd::string categoryAttributeValue;
+            reader.Read<AZStd::string>(categoryAttributeValue);
+            if (categoryAttributeValue.compare("display divider") == 0)
+            {
+                m_isSceneSetting = true;
+                setContentsMargins(8, 0, 4, 0);
+                setStyleSheet("QFrame {background-color: #333333; margin-top: 5px; border-top-right-radius: 2px; border-top-left-radius: 2px;}");
+            }
```

Comment:
Please comment this. If possible, see if the same thing can be applied using a style sheet applied from the asset inspector.